docs/source/GetStartedGuide.rst

```diff
@@ -46,36 +52,39 @@ how to switch to the ``avocado-lts`` repo.
 Finally, after deciding between regular Avocado releases or LTS, you
 can install the RPM packages by running the following commands::
 
-    sudo dnf install avocado
+    dnf install python-avocado
```

Comment:
Aren't we going to mention the install method from fedora repository? (just like you did in `README`, new lines 41 to 44).